### ISSUE TITLE
PSSH KID list is incorrect

### ISSUE THREAD

**Issue by skyhisi** [peaBerberian/isobmff-inspector#1]
I think at [line 50 of pssh.js][1] it's creating an array entry for each character in the KID string, rather than a single array element containing the whole KID string.

In AISOBMFFWVDFBUTFAII a single KID is displayed like:
```
 KID_count: 1
KIDs:
0: "±"
1: "V"
2: "Ï"
3: ""
4: "ù"
5: ""
6: "H"
7: "ô"
8: ""
9: "Ò"
10: ""
11: "-"
12: ")"
13: "¼"
14: "."
15: "¼"
```

Would also be more useful to display the KID and the PSSH data as hex.

[1]: https://github.com/peaBerberian/isobmff-inspector/blob/0bc36e7e9c58e74bb7cb87898fbf5a5950a6c705/src/boxes/pssh.js#L50

**Comment by peaBerberian** [comment]
You're right, thanks a lot (and sorry for the delay, I'm bad with keeping up with emails).
